CertifaiAI/classifai
# Sum v1.2 minor update + cache busting fix

# Description
v1.2 update:
- graphics update
- version update in scripts files and pom.xml

cache busting for static assets:
- using no-cache header, it will re-validate the static files served every time.

Testing for cache busting:
1. clear previous cache. Previous cache are still cached in browser, so clear it for the last time ever.
2. run classifai
3. open developer console and check the network, refresh and try to see the status 304 is there or not. (304 means validation performed, cached assets and the server one are the same. No update required)
![image](https://user-images.githubusercontent.com/76682536/108788312-a652ba00-75b2-11eb-95f2-733814fe16e7.png)
4. close classifai
5. Make some changes in the static assets file. I'll suggest changing this file, to get 
 all the response at the first page https://github.com/CertifaiAI/classifai/blob/main/classifai-core/src/main/resources/webroot/assets/i18n/landing-page-en.json. **Don't forget to save the changed file in order to reflect changes**
6. open classifai(using ./startclasifai.sh) and see changes made without clearing cache
![image](https://user-images.githubusercontent.com/76682536/108788806-ccc52500-75b3-11eb-899c-10f6379b90ad.png)


Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Tested on?


# Checklist:
